### DISCUSSION TITLE
Can users limit or turn off token pricing data for tags in twitter?

### DISCUSSION THREAD

**Discussion by cchamilt** [DimensionDev/Maskbook#6611] [Q&A] [answered]
Hi, I thought there was a way to turn off pricing data in twitter completely a few versions ago, but it seems I can't find it in settings.

If not, should there be at least a way to limit them with a good or bad list?  
It's getting obnoxious matching both # tags words that happen to match and $ that matches stock and practically any dollar amount to a token name.  Also, the data modal requires clicking on the screen to clear and many tweet margins, etc. in the twitter pages are links.

**Comment by Tedko** [comment]
good suggestion @wenluomask  we should add this back

**Comment by Tedko** [comment]
@cchamilt Thx

**Comment by Tedko** [comment]
@cchamilt you can do this --
<img width="1341" alt="Screen Shot 2022-07-15 at 18 20 25" src="https://user-images.githubusercontent.com/6687437/179318874-40ddc83a-3647-4387-94f0-d31d4531ee4b.png">
<img width="1341" alt="Screen Shot 2022-07-15 at 18 20 57" src="https://user-images.githubusercontent.com/6687437/179318877-b7223933-c48e-4f26-ad11-74930186ad9f.png">
Click the right top button

<img width="1341" alt="Screen Shot 2022-07-15 at 18 22 05" src="https://user-images.githubusercontent.com/6687437/179318890-1facb02e-a69e-419f-8c19-65b14e5b345a.png">
 turn the switch off

**Reply by Tedko** [reply]
switch on/off for `Market Trending` will also be added

**Reply by cchamilt** [reply]
Thanks, that does help!

**Comment by Tedko** [comment, marked as answer]
@wenluomask we should add `switch on/off` for `Market Trending` func as well

**Reply by wenluomask** [reply]
I have made a task issue to development team. Thanks for your advice @cchamilt !
